DataFormats/HcalDigi/interface/HcalDigiSoA.h

```diff
@@ -0,0 +1,39 @@
+#ifndef DataFormats_HcalDigi_HcalDigiSoA_h
+#define DataFormats_HcalDigi_HcalDigiSoA_h
+
+//TODO: Use Eigen column for data(?)
+//#include <Eigen/Core>
+//#include <Eigen/Dense>
+#include <array>
+
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+#include "DataFormats/HcalDigi/interface/HcalDigiCollections.h"
+
+namespace hcal {
+
+  using QIE11dataArray = std::array<uint16_t, QIE11DigiCollection::MAXSAMPLES>;
+  using QIE10dataArray = std::array<uint16_t, HBHEDataFrame::MAXSAMPLES>;
```

Comment:
Unfortunately `std::array` cannot be serialised.
Could you use `edm::StdArray` from `DataFormats/Common/interface/StdArray.h` instead ?

See `DataFormats/EcalDigi/interface/EcalDigiSoA.h` for an example.

Reply:
Done in https://github.com/cms-sw/cmssw/commit/a97492a732f4f78689de3d3212c4c7fcce5b0657